tests/unit/lib/test_helper_cluster.py

```diff
@@ -98,68 +81,46 @@ def test_auto_recompute_node_roles_in_cluster_6(self):
         self.assertEqual(computed_node_to_change.name, "data1")
         self.assertCountEqual(computed_node_to_change.roles, self.cm_roles)
 
-        # remove a voting only node
-        computed_node_to_change = ClusterTopology.node_with_new_roles(
-            [node for node in cluster_conf if node.name != "voting1"]
-        )
-        self.assertEqual(computed_node_to_change.name, "data1")
-        self.assertCountEqual(computed_node_to_change.roles, self.voting_only_roles)
-
         # remove a data node
         computed_node_to_change = ClusterTopology.node_with_new_roles(
             [node for node in cluster_conf if node.name != "data1"]
         )
         self.assertIsNone(computed_node_to_change)
 
-    def test_node_with_new_roles_in_cluster_5(self):
+    def test_auto_recompute_node_roles_in_cluster_5(self):
         """Test the automatic suggestion of new roles to an existing node."""
         cluster_conf = self.cluster_5_nodes_conf()
 
         # remove a cluster manager node
         computed_node_to_change = ClusterTopology.node_with_new_roles(
             [node for node in cluster_conf if node.name != "cm1"]
         )
-        self.assertTrue(computed_node_to_change.name.startswith("voting"))
         self.assertCountEqual(computed_node_to_change.roles, self.base_roles)
 
-        # remove a voting only node
-        computed_node_to_change = ClusterTopology.node_with_new_roles(
-            [node for node in cluster_conf if node.name != "voting1"]
-        )
-        self.assertEqual(computed_node_to_change.name, "voting2")
-        self.assertCountEqual(computed_node_to_change.roles, self.base_roles)
-
-        # remove a data node
-        computed_node_to_change = ClusterTopology.node_with_new_roles(
-            [node for node in cluster_conf if node.name != "data1"]
-        )
-        self.assertIsNone(computed_node_to_change)
-
```

Comment:
Is the data node test removed intentionally?

Reply:
Yes, there should be no "data" node in a "5 nodes cluster"